src/views/Tournament/Tournament.tsx

```diff
@@ -57,6 +57,63 @@ interface TournamentProperties {
     };
 }
 
+interface TournamentState {
+        new_tournament_group_id: number;
```

Comment:
Extra level of indent here

Reply:
Oops.  Sometimes I would copy the state initialization into the interface and modify so that's how that happened:

```
interface TournamentState {
    this.state = {  // cut this out after
        some_member: "Hi!",
    }
}
```